inlong-sort/sort-flink/sort-flink-v1.13/sort-connectors/tubemq/src/main/java/org/apache/inlong/sort/tubemq/table/TubeMQDynamicTableFactory.java

```diff
@@ -109,7 +109,7 @@ public DynamicTableSource createDynamicTableSource(Context context) {
         final DecodingFormat<DeserializationSchema<RowData>> valueDecodingFormat = getValueDecodingFormat(helper);
 
         // validate all options
-        helper.validate();
+        helper.validateExcept(INNERFORMATTYPE.toArray(new String[0]));
```

Comment:
maybe INNERFORMATTYPE should be a string to keep consistant with flink 1.15 ?

Reply:
fixed, thx